src/dmclock/sim/src/sim_client.h

```diff
@@ -253,7 +253,7 @@ namespace crimson {
 	    } // for
 	    ops_count += i.args.req_params.count;
 	  } else {
-	    assert(false);
+	    ceph_abort();
```

Comment:
This is a simulator to test and experiment with the dmclock algorithm. It is never compiled into ceph. In fact this change will prevent the simulator from compiling. Therefore this change needs to be removed